docs/seqcol.md

```diff
@@ -778,7 +777,7 @@ This is looser than strict identity, since we don't really care what the underly
 We also don't care about the order of the sequences.
 Instead, we need them to match level 1 digest of the `sorted_name_length_pairs` attribute.
 Thus, to assert that a BED file can be viewed for a particular genome, we compare the `sorted_name_length_pairs` digest of our reference genome with the sequence collection used to generate the file.
-There are only two possibilities for compatibility: 1) If the digests are equal, then the data file is directly compatible; 2) If not, we must check the `comparison` endpoint to see whether the `sorted_name_length_pairs` attribute of the sequence collection is a direct subset of the same array in the sequence collection attached to the genome browser instance.
+There are only two possibilities for compatibility: 1) If the digests are equal, then the data file is directly compatible; 2) If not, we must check the `comparison` endpoint to see whether the `name_length_pairs` attribute of the sequence collection is a direct subset of the same array in the sequence collection attached to the genome browser instance.
```

Comment:
Since `sorted_name_length_pairs` is now transient, it will not be available in the `comparison` endpoint.